README.md

```diff
@@ -1,24 +1,33 @@
+This repository was created as part of the <a href="https://utrechtuniversity.github.io/dataprivacyproject" target="_blank">Data Privacy Project</a>, a Research Data Management Support project at Utrecht University that ran between 2021 and 2023 and aimed to collect and improve information, tools and services surrounding personal data in research. It is  maintained by [Research Data Management Support](https://uu.nl/rdm). 
 
-Current overviews can be accessed here:
-- Mostly Python- and some R-based packages to generate [synthetic & simulated data](synthetic-simulated-data)
-- Python-, R-based and user interface tools to [deidentify data](deidentification)
-- Software to [securely analyse data](secure-computing), e.g., to bring code-to-data or encrypt data. 
+We currently have the following overviews:
+
+- Packages and (open source) software to [de-identify (pseudonymise/anonymise) data](deidentification)
+- Packages and services to generate [synthetic & simulated data](synthetic-simulated-data)
+- Software to [securely analyse or encrypt data](secure-computing)
+
+## Why this repository?
+
+There are many existing tool overviews. However, many of them are generic, and are therefore not always very useful for scientific researchers. Sometimes, existing tool overviews mention many commercial service providers which are out of reach or simply not permitted for use in research that involves personal data. Therefore, in this repository, our focus is mostly on open source tools and graphical user interfaces which are suited for use in research (and thus not only research performed at Utrecht University).
 
 ## Disclaimer
 
 Important notes:
+
 1. In no way do we claim that these overviews are complete or fully accurate. Please let us know if we are missing important tools and/or if an existing entry needs correction.
-2. We are not responsible for the quality of the tools in this overview. Moreover, the quality of the results when using these tools is largely dependent on your own use of the tool. We recommend to use these tools responsibly and be wary of unrealistic promises that tools may make. If a tool we included turns out to be of low quality, please let us know so we can take note of that. 
+2. We are not responsible for the quality of the tools in this overview. Moreover, the quality of the results when using these tools is largely dependent on your own use of the tool. We recommend using these tools responsibly and being wary of unrealistic promises that tools may make. If a tool we included turns out to be of low quality, please let us know so we can take note of that or remove it from our overview. 
 
 ## License and citation
 
-This repository is licensed under an MIT license. You can [view the license text here](/LICENSE.md). 
+This repository is licensed under an MIT license. You can [view the license here](/LICENSE.md). 
```

Comment:
What about putting the link on the "MIT license" text instead?

Reply:
Check!